Rename context swapping to context extension. Fix some grammar issues… (#4196)
trpc/trpc: www/docs/server/middlewares.md

});
```

## Context Extension

> "Context Extension" in tRPC is a very powerful feature of middlewares, where you can dynamically add and override keys on a base procedure's context in a typesafe manner.

Reviewer: ```suggestion
"Context Extension" enables middlewares dynamically add and override keys on a base procedure's context in a typesafe manner.
```

> Below we have an example of a middleware that changes properties of a context, the change(s) are then available to all chained consumers, such as other middlewares and procedures:

Reviewer: ```suggestion
Below we have an example of a middleware that changes properties of a context, the changes are then available to all chained consumers, such as other middlewares and procedures:
```

"properties" is already plural and I think it's clear enough that multiple changes and a singular change will function in the same way
